ERCS/erc-7007.md

```diff
@@ -133,6 +139,10 @@ ERC-7007 Metadata JSON Schema for reference
         "aigc_data": {
             "type": "string",
             "description": "A URI pointing to a resource with mime type image/* representing the asset to which this AIGC NFT represents."
+        },
+        "verificaton": {
+            "type": "string",
+            "description": "Specify the verification method. Input zkML or opML."
```

Comment:
I think this makes sense. want to propose another possible wording:
"proof_type": {
  "type": "validity" / "fraud",
  ...
}
the point here is to align with "aigc_type".
we can discuss this.